client/python/examples/README.md

```diff
@@ -0,0 +1,185 @@
+# Python Client Examples
+
+Currently there are three example files:
+
+- [simple.py](#simple.py)
+- [general.py](#general.py)
+- [async_logging.py](#async_logging.py)
+
+There are also some codeblocks with useful code snippets:
+
+- [Using Basic Auth](#using-basic-auth)
+
+
+## Setup
+
+Please see the [python client docs](https://github.com/G-Research/armada/blob/master/client/python/README.md) for getting the client setup.
+
+### Running the examples
+
+Each example has three enviromental variables for setup
+
+```bash
+export ARMADA_SERVER=localhost
+export ARMADA_PORT=443
+export DISABLE_SSL=true
+```
+
+You can then simply run one of the examples
+
+```bash
```

Comment:
Are you missing an explanation of how to create a python virtual environment and how to install client to an environment?

Reply:
True but if any tries to copy your code in this PR it will fail without any notice of why that is the case?

Reply:
I get what you mean, I would assume that if a user is checking the example README they would already have it installed, but I understand that this is not always the case. I will link to https://github.com/G-Research/armada/blob/master/client/python/README.md directly as a guide for installing?